code/modules/events/slaughterevent.dm

```diff
@@ -0,0 +1,69 @@
+
```

Comment:
Again, may want to forego the question for this one--but that's just my opinion. Did you happen to test this one out?